tests/components/kem/test_sensor.py

```diff
@@ -0,0 +1,83 @@
+"""Tests for the KEM sensors."""
+
+from __future__ import annotations
+
+from unittest.mock import AsyncMock
+
+from aiokem import AioKem
+from freezegun.api import FrozenDateTimeFactory
+from syrupy import SnapshotAssertion
+
+from homeassistant.components.kem.const import SCAN_INTERVAL_MINUTES
+from homeassistant.const import STATE_UNAVAILABLE
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import entity_registry as er
+
+from tests.common import MockConfigEntry, async_fire_time_changed, snapshot_platform
+
+DISABLED_ENTITIES = [
+    "sensor.generator_1_total_operation",
+    "sensor.generator_1_total_runtime",
+    "sensor.generator_1_runtime_since_last_maintenance",
+    "sensor.generator_1_device_ip_address",
+    "sensor.generator_1_server_ip_address",
+]
+
+
+async def test_sensors(
+    hass: HomeAssistant,
+    platform_sensor,
+    entity_registry: er.EntityRegistry,
+    snapshot: SnapshotAssertion,
+    kem_config_entry: MockConfigEntry,
+    mock_kem: AioKem,
+    freezer: FrozenDateTimeFactory,
+) -> None:
+    """Test the KEM sensors."""
+
+    # Enable the disabled entities
+    for entity_id in DISABLED_ENTITIES:
+        entity_registry.async_update_entity(entity_id=entity_id, disabled_by=None)
+
+    # Move time to next update
+    freezer.tick(SCAN_INTERVAL_MINUTES)
+    async_fire_time_changed(hass)
+    await hass.async_block_till_done(wait_background_tasks=True)
```

Comment:
Why do we do this?

Reply:
i think it was to get the disabled sensors to update. but that's not needed with the new fixture